src/invokeArgs.js

```diff
@@ -0,0 +1,35 @@
+import { curryN } from 'ramda';
+
+/**
+ * Invokes the method at path of object with given arguments.
+ *
+ * @func invokeArgs
+ * @memberOf RA
+ * @since {@link https://char0n.github.io/ramda-adjunct/2.27.0|v2.27.0}
+ * @category Object
+ * @sig Array -> Array -> Object -> *
+ * @param {Array.<string|number>} path The path of the method to invoke
+ * @param {Array} args The arguments to invoke the method with
+ * @param {Object} obj The object to query
+ * @return {*}
+ * @example
+ *
+ * RA.invokeArgs(['abs'], [-1], Math); //=> 1
+ * RA.invokeArgs(['path', 'to', 'non-existent', 'method'], [-1], Math); //=> undefined
+ */
+
+const invokeArgs = curryN(3, (pathToMethod, args, obj) => {
```

Comment:
I was thinking about how to simplify this code yesterday. The pattern of simplification that came to my mind is following: 
1.) pluck method from a path
2.) pluck a parent context
3.) if method is a function we bind the parent context and apply the arguments (if not we return undefined)
4.) if the method call itself throws, we should rethrow

Pseudocode (just to demonstrate the idea)
```js
const invokeArgs = curryN(3, (mpath, args, obj) => {
  const method = path(mpath, obj);
  const context = path(init(mpath), obj);
  
  if (isNotFunction(method)) return undefined;
  if (isEmptyArray(mpath)) return undefined;
  
  const boundMethod = bind(method, context);
  return apply(boundMethod, args);
});
```